infrastructure/variables.tf

```diff
@@ -46,7 +46,7 @@ variable "vault_section" {
 # region app config
 
 variable "idam_api_url" {
-  default = "http://betaDevBccidamAppLB.reform.hmcts.net"
+  default = "http://idam-api-idam-saat.service.core-compute-saat.internal"
```

Comment:
Should sandbox instance also talk to this saat one?

Reply:
for now, :)